ouroboros-network/src/Ouroboros/Network/BlockFetch.hs

```diff
@@ -178,17 +179,24 @@ data BlockFetchConsensusInterface peer header block m =
        -- fragment, the filtering (with a linear cost) is stopped as soon as a
        -- block has a slot number higher than this slot number, as it cannot
        -- have been downloaded anyway.
-       readFetchedMaxSlotNo    :: STM m MaxSlotNo,
+       readFetchedMaxSlotNo   :: STM m MaxSlotNo,
 
        -- | Given the current chain, is the given chain plausible as a
        -- candidate chain. Classically for Ouroboros this would simply
        -- check if the candidate is strictly longer, but for Ouroboros
        -- with operational key certificates there are also cases where
        -- we would consider a chain of equal length to the current chain.
        --
-       plausibleCandidateChain :: HasCallStack
+       filterCandidates        :: HasCallStack
```

Comment:
Can we also drop `compareCandidateChains`? We're thinking it should be dead-code as of this PR.